apps/workspace-host/src/interface.js

```diff
@@ -26,6 +26,7 @@ import { logger } from '@prairielearn/logger';
 import * as sqldb from '@prairielearn/postgres';
 import * as Sentry from '@prairielearn/sentry';
 import * as workspaceUtils from '@prairielearn/workspace-utils';
+import { cache } from '@prairielearn/cache';
```

Comment:
You should add this package to `apps/workspace-host/package.json`.

Reply:
Added to this package to `apps/workspace-host/package.json` in [eae6537](https://github.com/PrairieLearn/PrairieLearn/pull/9408/commits/eae65374a2f77e98f4937bcd4537406b04524243).